subjects/fishandchips/README.md

```diff
@@ -0,0 +1,40 @@
+## Fishandchips
+
+### Instructions
+
+Write a function called `Fishandchips()` Write a function that takes an integer and returns a string.
```

Comment:
there is `Write a function` two time at the line

Reply:
it's fixed